code/src/main/kotlin/com/expediagroup/sdk/core/model/exception/service/ExpediaGroupApiException.kt

```diff
@@ -1,23 +1,20 @@
-/*
- * Copyright (C) 2022 Expedia, Inc.
```

Comment:
I think it's removed by accident? let's bring it back please (applicable to any class with missing copyright strings)